README.md

```diff
@@ -20,9 +20,390 @@ Or install it yourself as:
 
     $ gem install cassia-ruby
 
-## Usage
+## Configuration
+Here is a list of the available configuration options and their default values
 
-TODO: Write usage instructions here
+| Option          | Description                                                                                                                                                                                                                                                | Default Value                                |
+|-----------------|------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------|----------------------------------------------|
+| `ac_url`        | The URL of your AC server address                                                                                                                                                                                                                                  | `"http://iteratelabs.cassia.pro"` |
+| `client_id`     | Your Cassia API Developer Key                                                                                                                                                                                                                                  | `iteratelabs`                                          |
+| `secret`       | Your Cassia API Developer Secret                                                                                                                                                           |
+ `7d8b99e989344b6f` |
```

Comment:
I don't think we want to include this here, also I think you have an extra `|` which is adding a new row